core/src/main/java/com/graphhopper/GraphConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.graphhopper;
+
+import com.graphhopper.storage.DAType;
+
+public final class GraphConfig {
```

Comment:
I like the idea of grouping together related configuration parameters. I did the same with `RoutingConfig` recently, maybe we can already do this in master.